### ISSUE TITLE
Build failure with 7.7.162

### ISSUE THREAD

**Issue by JanMarvin** [jeroen/V8#63]
Build fails with `V8 version 7.7.162` due to https://github.com/v8/v8/commit/9ae5a9a1d6920da424647dd876b79a28c2cf63df#diff-3dd7a0f46b1e5c1cc67921449e7180ac

```{C++}jmg@seagull> R CMD build /home/jmg/Source/V8                                                                                                                                                 ~/Source/v8
* checking for file ‘/home/jmg/Source/V8/DESCRIPTION’ ... OK
* preparing ‘V8’:
* checking DESCRIPTION meta-information ... OK
* cleaning src
* running ‘cleanup’
* installing the package to build vignettes
      -----------------------------------
* installing *source* package ‘V8’ ...
** using staged installation
Using PKG_CFLAGS=-I/usr/include/v8 -I/usr/include/v8-3.14
Using PKG_LIBS=-lv8 -lv8_libplatform
** libs
rm -f V8.so RcppExports.o bindings.o
g++ -std=gnu++11 -I"/usr/include/R/" -DNDEBUG -I/usr/include/v8 -I/usr/include/v8-3.14 -I"/home/jmg/R/x86_64-pc-linux-gnu-library/3.6/Rcpp/include" -D_FORTIFY_SOURCE=2 -fvisibility=hidden -fpic  -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt  -c RcppExports.cpp -o RcppExports.o
g++ -std=gnu++11 -I"/usr/include/R/" -DNDEBUG -I/usr/include/v8 -I/usr/include/v8-3.14 -I"/home/jmg/R/x86_64-pc-linux-gnu-library/3.6/Rcpp/include" -D_FORTIFY_SOURCE=2 -fvisibility=hidden -fpic  -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt  -c bindings.cpp -o bindings.o
bindings.cpp: In function ‘ctxptr make_context(bool)’:
bindings.cpp:246:55: error: no matching function for call to ‘v8::Object::Set(v8::Local<v8::String>&, v8::Local<v8::Object>)’
  246 |     context->Global()->Set(console, console_template());
      |                                                       ^
In file included from V8_types.h:1,
                 from bindings.cpp:2:
/usr/include/v8.h:3323:37: note: candidate: ‘v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, v8::Local<v8::Value>, v8::Local<v8::Value>)’
 3323 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context,
      |                                     ^~~
/usr/include/v8.h:3323:37: note:   candidate expects 3 arguments, 2 provided
/usr/include/v8.h:3326:37: note: candidate: ‘v8::Maybe<bool> v8::Object::Set(v8::Local<v8::Context>, uint32_t, v8::Local<v8::Value>)’
 3326 |   V8_WARN_UNUSED_RESULT Maybe<bool> Set(Local<Context> context, uint32_t index,
      |                                     ^~~
/usr/include/v8.h:3326:37: note:   candidate expects 3 arguments, 2 provided
make: *** [/usr/lib64/R/etc/Makeconf:176: bindings.o] Error 1
ERROR: compilation failed for package ‘V8’
* removing ‘/tmp/RtmpSABVf8/Rinst4ea95f169060/V8’
      -----------------------------------
ERROR: package installation failed
```

I'll have a look after work.

**Comment by JanMarvin** [comment]
```{C++}
    if(context->Global()->Set(context, console, console_template()).IsNothing())
      Rcpp::warning("Could not set console.");
```
Quick fix

**Comment by JanMarvin** [comment]
Realized, that I misinterpreted that line. The output should be printed.

**Comment by jeroen** [comment]
Thanks! Let me try to reproduce this...

**Comment by jeroen** [comment]
Fixed on CRAN now in V8 version 2.3.
